### ISSUE TITLE
CMS verify MDM Signed Message: pkcs7: signing time 2024-07-11T08:08:45Z is outside of certificate validity 2024-07-11T08:08:48Z to 2028-07-11T08:08:48Z

### ISSUE THREAD

**Issue by ppdouble** [micromdm/micromdm#1015]
### What version of `micromdm` are you using?

v1.11.0

### What `micromdm` command did you run?

The steps that user did is:

1. Download(manually) a enroll profile which is already signed
2. Install(manually) the profile to enroll his/her device
3. Then the device will communicate to micromdm

### What did you expect to see?

Everything works fine as usuall

### What did you see instead?

CMS verify MDM Signed Message: pkcs7: signing time 2024-07-11T08:08:45Z is outside of certificate validity 2024-07-11T08:08:48Z to 2028-07-11T08:08:48Z

Question: Is this flag [MICROMDM_DEVICE_SIGNATURE_SKEW](https://github.com/micromdm/micromdm/blob/v1.12.1/cmd/micromdm/serve.go#L131) solve this kind of problem of enroll stage?

Related: 

https://github.com/micromdm/micromdm/issues/886

https://github.com/micromdm/micromdm/pull/871

https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-10/security/threat-protection/security-policy-settings/maximum-tolerance-for-computer-clock-synchronization

https://github.com/micromdm/micromdm/pull/887

**Comment by jessepeterson** [comment]
Yes, the device noted its signature as 3 seconds before the certificate was issued. This usually means either the server issuing certs, or the device's time is incorrect. Solutions are:

* Ensure the time is correct on the device or
* Ensure the time is correct on the CA issuing server or
* Get your CA to issue certificates that have an earlier issuance time or
* Wait for smallstep/pkcs7#21 to get a fix and for us to decide to engineer a fix based on it

Note: The skew time is just for the allowance of signature skew, the error is generated because the signature is outside of the certificate validity (for which the skew switch does not help).
